### ISSUE TITLE
Модель ветвления

### ISSUE THREAD

**Issue by igsekor** [Y-Doka/y-doka.site#583]
В документации проекта не описана модель ветвления. Вывода сделал по прочтении трех файлов:

- [README.md](https://github.com/Y-Doka/y-doka.site/blob/master/README.md)
- [CONTRIBUTING.md](https://github.com/Y-Doka/y-doka.site/blob/master/CONTRIBUTING.md)
- [WRITING.md](https://github.com/Y-Doka/y-doka.site/blob/master/WRITING.md)

**Comment by pepelsbey** [comment]
Модель ветвления чего? :)

**Comment by igsekor** [comment]
Модель ветвления в репозитории. Есть ветка `master`. Если пользоваться подходом GitHub Flow, то этого вполне достаточно. Есть ветки для исправления багов, фич и прочее. Похоже на GitFlow. В документации нет определенности  с этим вопросом.

**Comment by solarrust** [comment]
Кажется, речь о соглашении по именованию веток.
У нас так:
- Если пишите статью для контента сайта — `article/figcaption` — `article или longread/тема-статьи`
- Если ветка с фичей — `feature/name-of-feature`
- Для хотфиксов без ишью — `fix/name-of-bug`
- Для редакторских правок — `editor/issue` или `editor/article-name`

Пока эта инфа хранится вне гитхаба, но мы её обязательно перенесём. Спасибо!

@furtivite загляни сюда.

**Comment by igsekor** [comment]
Да. Именно об этом. И о необходимости использовать данный подход.

**Comment by furtivite** [comment]
@igsekor, привет, спасибо, что обратил внимание. Дополним документацию. 

Мы сейчас действительно живём примерно по этому варианту. @solarrust @pepelsbey сделаю пиар с этими вставками.
